### DISCUSSION TITLE
update payment method error

### DISCUSSION THREAD

**Discussion by Assone** [community/community#49218] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi, I am unable to update my payment method. Can you tell me what to do?
<img width="1155" alt="image" src="https://user-images.githubusercontent.com/14324389/222949911-7de7f66c-dc90-4134-a00f-32bacc7c7b41.png">

**Comment by jge162** [comment]
Hello @Assone A admin here might be able to assist you. Or you can reach out to billing [support](https://support.github.com/contact/account?tags=hubberfy_billing_and_payments) if you have not already. Bests

**Reply by Assone** [reply]
hi @jge162 ，I did submit a tickets, but no one replied to me.

**Reply by jge162** [reply]
How long ago did you submit ticket? I’ve seen people wait up to 8 days. It’s based on FIFO. Or in other words you are in line until. Your turn comes up.

**Reply by Assone** [reply]
Well, about five days ago, it would have been nice if it didn't take too long to process. Since I have an autopay, hopefully I can update the payment method before the next billing date.

**Reply by queenofcorgis** [reply]
Appreciate your patience - tickets are answered based on our team's resources and the # of tickets ahead of you. Wanted to share, you are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
